packages/compiler-cli/ngcc/src/packages/configuration.ts

```diff
@@ -27,6 +32,11 @@ export interface NgccPackageConfig {
    * will be absolute.
    */
   entryPoints: {[entryPointPath: string]: NgccEntryPointConfig;};
+  /**
+   * A collection of regexes that match deep imports to ignore, for this package, rather than
```

Comment:
```suggestion
   * A collection of regexes that match deep imports to ignore, for this project, rather than
```
This seems to be a project level configuration, not a package one?

Reply:
Actually it is at the package level. You define the regexes for each package that will do the importing. This way we can isolate turning off warnings only to specific packages.